pyo3-macros-backend/src/attributes.rs

```diff
@@ -43,6 +43,19 @@ impl Parse for NameAttribute {
     }
 }
 
+/// For specifying the path to the pyo3 crate.
+#[derive(Clone, Debug, PartialEq)]
+pub struct PyO3PathAttribute(pub Path);
```

Comment:
Now that we have `crate =` I'd be nice to call this `PyO3CrateAttribute` or `CrateAttribute`, similarly some variables called `pyo3_path` might want to be called `pyo3_crate` (or `krate`?! - then expanding it inside proc-macros as `#krate` is no longer than macro_rules `$crate`)

Reply:
Agreed, done!